app/controllers/admin/npq/applications/notes_controller.rb

```diff
@@ -4,36 +4,40 @@ module Admin
   module NPQ
     module Applications
       class NotesController < Admin::BaseController
-        skip_after_action :verify_authorized
         skip_after_action :verify_policy_scoped
 
-        def edit
-          @npq_application = NPQApplication.find(params[:id])
-        end
+        before_action :assign_npq_application
+
+        def edit; end
 
         def update
-          npq_application = NPQApplication.find(params[:id])
-          npq_application.assign_attributes(note_params)
+          @npq_application = NPQApplication.find(params[:id])
```

Comment:
Same as previous, do we need this duplicate loading with assign_npq_application in place?